lib/testing/src/lib/core/actions/identity/tasks.service.ts

```diff
@@ -73,7 +76,15 @@ export class TasksService {
         } catch (error) {
             Logger.error('Complete Task - Service error, Response: ', JSON.parse(JSON.stringify(error)).response.text);
         }
+    }
 
+    async completeTaskOfAProcess(processId: string, appName: string): Promise<any> {
+        try {
+            const task = await this.queryService.getProcessInstanceTasks(processId, appName);
```

Comment:
Are you sure that this method returns only the active tasks? Furthermore, you could have more than one task. In this case, you should complete all of them, right?